src/coreclr/nativeaot/Runtime/CommonMacros.h

```diff
@@ -62,7 +62,11 @@
 #endif
 #endif
 
-#ifndef __GCENV_BASE_INCLUDED__
+// TODO: gcenv.base.h provides these definitions only if it has been included and windows.h
```

Comment:
What is forcing these?

I tried backing this change out in https://github.com/dotnet/runtime/commit/3aafff4945fa894fce3440fde8381b18f25d4554 and it still builds.